modules.d/90livenet/livenetroot.sh

```diff
@@ -6,21 +6,33 @@ type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
 . /lib/url-lib.sh
 
 PATH=/usr/sbin:/usr/bin:/sbin:/bin
+RETRIES={RETRIES:-100}
+SLEEP={SLEEP:-5}
```

Comment:
Without a `$` won't these variables expand to a literal `{RETRIES:-100}` and `{SLEEP:-5}` and not defaults of `100` and `5`?